lib/colorls/core.rb

```diff
@@ -246,6 +268,7 @@ def fetch_string(path, content, key, color, increment)
 
       [
         long_info(path, content),
+        git_info(path,content),
```

Comment:
How about colorizing the output of `git_info`? White if '-', Green if 'A', Yellow if 'C', Red if 'U'.

Reply:
sure :+1: